Code/GraphMol/Wrap/rough_test.py

```diff
@@ -5602,7 +5602,9 @@ def testForwardEnhancedStereoGroupIds(self):
 
     stgs[0].SetWriteId(7)
     self.assertEqual(stgs[0].GetWriteId(), 7)
-    self.assertEqual(Chem.MolToCXSmiles(m), 'C[C@H](O)Cl |o7:1|')
+    wp = Chem.SmilesWriteParams()
+    # wp.rigorousEnhancedStereo = False
```

Comment:
This one is not a new test, but changes to make the existing test work as before when the rigorousEnhancedStereo  was the default.  Now that it is not the default, this can be removed (but it has no effect as written).  I will remove this change.